docs/subnets/customize-a-subnet.md

```diff
@@ -440,6 +441,92 @@ In addition to the AllowList interface, the FeeConfigManager adds the following
 
 Subnet-EVM contains example contracts for precompiles under `/contract-examples`. It's a hardhat project with tests, tasks. For more information see [contract examples README](https://github.com/ava-labs/subnet-evm/tree/master/contract-examples#subnet-evm-contracts).
 
+## Network Upgrades: Enable/Disable Precompiles
+
+:::warning
+
+Performing a network upgrade requires coordinating the upgrade network-wide. A network upgrade changes the rule set used to process and verify blocks, such that any node that upgrades incorrectly or fails to upgrade by the time that upgrade goes into effect may become out of sync with the rest of the network.
+
+Any mistakes in configuring network upgrades or coordinating them on validators may cause the network to halt and recovering may be difficult.
+
+:::
+
+In addition to specifying the configuration for each of the above precompiles in the genesis chain config, they can be individually enabled or disabled at a given timestamp as a network upgrade. Disabling a precompile disables calling the precompile and destructs its storage so it can be enabled at a later timestamp with a different configuration if desired.
```

Comment:
```suggestion
In addition to specifying the configuration for each of the above precompiles in the genesis chain config, they can be individually enabled or disabled at a given timestamp as a network upgrade. Disabling a precompile disables calling the precompile and destructs its storage so it can be enabled at a later timestamp with a news configuration if desired.
```